frontend/src/components/design-system/containers/PillComponent.vue

```diff
@@ -0,0 +1,17 @@
+<script setup lang="ts">
+import LabelText from '../typography/LabelText.vue';
```

Comment:
Ještě tohle, asi takový Vue best practice?

```suggestion
import LabelText from '@/components/typography/LabelText.vue';
```

Reply:
Na to jsem úplně zapomněla, že se používá. Opraveno.